### PR TITLE
Clone of the New York times page

### PR DESCRIPTION
**Pull request by tacodtripe**
# Clone of the New York Times article

In this project we:
- Used grid, float and flex positioning in order to create the most accurate layout from the original page.
- Exercised images and video embedding.
- Combined different CSS styles to make it look professional.

### CODE REVIEW

**Comment by majovanilla** [comment]
Hey team, 👋

## You did an *amazing*🌟 ✨ job on this project! 👊🏻  💯

- Your project looks like a very professional clone of the original page.
- Good Readme template.

There are still some changes 👇 you need to implement to approve this request 🔍

### Required Changes: ✍️

- Always remember to check your HTML using the [validator](https://validator.w3.org/). You have several errors and warnings. Please fix them.

- Improve your Readme file. It should be more descriptive of the project: what you did, how you did it, why you did it... You are using a great template!

- Please write meaningful pull request messages. Check this [example](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/)

- The NYT logo should be a link.

- Your main title should be centered like the original page.

- Your bottom section needs some work. The articles' titles are overlapping with the image below and need a little more space (margin or padding) to make them look better:
Screenshot from 2020-03-04 16-48-26

Original page:
Screenshot from 2020-03-04 16-49-10

- You are not using float, grid or flex to position the text in the Theory of inflation image. This is a very important requirement in both Microverse and Odin Project since that is the main objective of the project. Please check the comments on the CSS code that I made below and change that part.


### Optional Changes:

If you want to make your project even better 🔥💥, I have a few suggestions you can implement.  

These are completely [Optional]: 🦔

- Use semantic HTML in the structure of your HTML file. Favor the use of `<nav>`,  `<main>`,  `<footer>`, to enclose the elements on your file. You are already using them, but you should not enclose the header and the footer in the main tag.

- You could improve the look of your page by implementing the hover styling on the video and text links.

## Please, make all REQUIRED changes and submit your request again! 😀

If you have any questions, feel free to send me a message through Slack (@majo)
I would be happy to help! 👩‍💻 


Happy coding 🤓

--Maria Reyes

File: style.css [hunk 1]
```diff
@@ -0,0 +1,741 @@
...
+
+.no-background {
+  background-color: transparent !important;
+}
+
+.no-background i {
+  font-size: 25px !important;
+  color: #666 !important;
+}
+
```
Comment (majovanilla) [inline style.css]: Please avoid the use of !important as much as posible!

File: style.css [hunk 2]
```diff
@@ -0,0 +1,741 @@
...
+#r2c3 {
+  top: 64%;
+  left: 50%;
+}
+
+#r2c4 {
+  top: 64%;
+  left: 66%;
+}
```
Comment (majovanilla) [inline style.css]: Use grid, flex or float to position the text above the image

File: style.css [hunk 3]
```diff
@@ -0,0 +1,741 @@
...
+  width: 100%;
+  height: 40vh;
+}
+
+#video-information {
+  color: #666;
+  line-height: 1.2 !important;
+  width: 100% !important;
+  font-size: 1rem !important;
+}
```
Comment (majovanilla) [inline style.css]: avoid the use of !important

File: style.css [hunk 4]
```diff
@@ -0,0 +1,741 @@
...
+  margin-left: 10%;
+}
+
+#bicep2-information {
+  color: #666;
+  font-size: 0.9rem !important;
+  line-height: 1.2 !important;
+  margin-bottom: 3em !important;
+}
+
```
Comment (majovanilla) [inline style.css]: avoid the use of !important

File: index.html [hunk 5]
```diff
@@ -0,0 +1,505 @@
...
+                        <a href="#">
+                            Create an <span id="bottom">account </span>or log in
+                        </a>
+                    </p>
+                </div>
+                <aside>
+                    <h4>Editor's Picks</h4>
+                    <hr>
+                    <div>
+                        <img src="https://static01.nyt.com/images/2020/02/27/fashion/24MASSAGE1-promo/24MASSAGE1-promo-square640-v2.jpg?quality=75&auto=webp&disable=upscale&width=350" alt="" width="75" height="75">
```
Comment (euqueme) [inline index.html]: - [ ] the image on line #86 shouldn't have style selectors (width and height should be in a class or id in the CSS file)

File: index.html [hunk 6]
```diff
@@ -0,0 +1,505 @@
...
+                        <p class="picks">
+                            <a href="#">
+                                The Secret to Beauty: A Stranger’s Hands Inside Your Mouth
+                            </a>
+                        </p>
+                    </div>
+                    <div>
+                        <img src="https://static01.nyt.com/images/2020/02/23/obituaries/23bsmith/23bsmith-square640.jpg?quality=75&auto=webp&disable=upscale&width=350" alt="" width="75" height="75">
```
Comment (euqueme) [inline index.html]: - [ ] the image on line #94 shouldn't have style selectors (width and height should be in a class or id in the CSS file)

File: index.html [hunk 7]
```diff
@@ -0,0 +1,505 @@
...
+                        <p class="picks">
+                            <a href="#">
+                                The Secret to Beauty: A Stranger’s Hands Inside Your Mouth
+                            </a>
+                        </p>
+                    </div>
+                    <div>
+                        <img src="https://static01.nyt.com/images/2020/02/26/nyregion/harlem-gator/harlem-gator-thumbLarge.jpg" alt="" width="75" height="75">
```
Comment (euqueme) [inline index.html]: - [ ] the image on line #102 shouldn't have style selectors (width and height should be in a class or id in the CSS file)

File: index.html [hunk 8]
```diff
@@ -0,0 +1,505 @@
...
+                        <p class="picks">
+                            <a href="#">
+                                The Secret to Beauty: A Stranger’s Hands Inside Your Mouth
+                            </a>
+                        </p>
+                    </div>
+                    <div>
+                        <img src="https://static01.nyt.com/images/2020/03/01/business/28velvetrope-illo/28velvetrope-illo-square640.jpg?quality=75&auto=webp&disable=upscale&width=350" alt="" width="75" height="75">
```
Comment (euqueme) [inline index.html]: - [ ] the image on line #110 shouldn't have style selectors (width and height should be in a class or id in the CSS file)

**Review by euqueme — CHANGES_REQUESTED** [review]
> ### Pull Request under the review by Maria Eugenia Queme P. (TSE)

### Hello @tacodtripe and @Cathella  :balloon: :lollipop: 

### Thank you for doing the changes requested by the previous reviewer :medal_sports: :medal_sports: your project looks excellent, and the page looks almost like the original :star: and I'm sure you won't have any problems passing in the next iteration, but please address the changes I've put in your code bellow and next :point_down: 

## Section specific

#### CSS bad practices

- [ ] no inline styling. please remove the inline styling I spotted in your HTML file, all style selectors should be declare in IDs or Classes within the CSS file

- [ ] no repeating of the same rule, don't repeat the same selector in all the classes instead declare them in chain or put them in a class these are repeating multiple times:

- display: flex; (21 times)
- align-items: center; (8 times)
- flex-direction: column; (9 times)
- width: 100%; (5 times)
- justify-content: space-between; (4 times)

### That's it, team! :raised_hands: 

### Please :pray: do the changes :woman_technologist: :man_technologist: and submit back for a code review when you are done :memo: 

Happy coding! :rocket: 

Any question? ask me on slack **@ Maru**

PROJECT STATUS: REQUESTED CHANGES :traffic_light:

**Comment by lekegitrepo** [comment]
### This PR is under Review by [TSE] _Temitayo Adeleke_

**Comment by lekegitrepo** [comment]
### Good job 👏  👏  for implementing the required changes.
**_The project looks great now_** 💯
### _This project is approved. Wish you all the best with your next project._

### Reviewed by: _Temitayo Adeleke_

-   Github: [@lekegitrepo](https://github.com/lekegitrepo)
-   Twitter: [@leketemi](https://twitter.com/leketemi)
-   Linkedin: [linkedin](https://www.linkedin.com/in/adeleke-temitayo-a69125188/)

**Review by lekegitrepo — APPROVED** [review]
## Status: Approved

**Pull request merged**